### PR TITLE
Seqan tools

### PR DESCRIPTION
**Pull request by gurgese**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/seqan_tcoffee/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  name: seqan_tcoffee
+  version: '1.13.3'
+
+source:
+  fn: seqan-src-2.1.1.tar.gz
+  url: http://packages.seqan.de/seqan-src/seqan-src-2.1.1.tar.gz
+  sha256: df083d5939d7e73fd00bbc6bb291ac7ea7b8d446cf89a9758fdffdc0eca9ec1c
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/seqan_tcoffee/meta.yaml]: This one should start with build number 0.

File: recipes/seqan_tcoffee/build.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+mkdir -p build
+cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release
```
Comment (bgruening) [inline recipes/seqan_tcoffee/build.sh]: A general question. You also could use the `-DCMAKE_INSTALL_PREFIX` I assume?

File: recipes/micro_razers/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  name: micro_razers
+  version: '1.0.6'
+
+source:
+  fn: seqan-src-2.1.1.tar.gz
+  url: http://packages.seqan.de/seqan-src/seqan-src-2.1.1.tar.gz
+  sha256: df083d5939d7e73fd00bbc6bb291ac7ea7b8d446cf89a9758fdffdc0eca9ec1c
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/micro_razers/meta.yaml]: Start with 0, if you add a new package/version.

File: recipes/seqan_tcoffee/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+package:
+  name: seqan_tcoffee
```
Comment (yhoogstrate) [inline recipes/seqan_tcoffee/meta.yaml]: Why is there a separate seqan_tcoffee and seqan-library?

File: recipes/seqan_tcoffee/meta.yaml (hunk 1 above)
Comment (gurgese) [reply recipes/seqan_tcoffee/meta.yaml]: Yes, I see you have already changed it :+1:

File: recipes/seqan_tcoffee/build.sh (hunk 2 above)
Comment (gurgese) [reply recipes/seqan_tcoffee/build.sh]: I think so, 
I can update line 6 with -DCMAKE_INSTALL_PREFIX=$PREFIX/bin and test it.

Comment (bgruening) [reply recipes/seqan_tcoffee/build.sh]: Would be great if you can test this! Thanks!

Comment (gurgese) [reply recipes/seqan_tcoffee/build.sh]: Dear @bgruening, do you have other tests or can we merge this PR?

**Comment by bgruening** [comment]
Sorry, I have dropped the ball here. Thanks for the ping!

**Pull request merged**